.github-scheduled-workflows/publish-main.yaml

```diff
@@ -1,4 +1,4 @@
-name: Publish to main 
+name: Post Release - Update CSV
```

Comment:
I don't think we can have the following:

```yaml
  workflow_dispatch:
    inputs:
      ref:
        description: "The Operator branch/tag to checkout when creating the image."
        required: true
        default: "release_operator_image"
      prev_version:
        description: "The Operator version to replace"

env:
  GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }} 
  release-version: ${{ github.event.inputs.ref }}
  replace_version: ${{ github.event.inputs.prev_version }}
```

Re-declaring `release-version` will overwrite the env variable inherited from `release.yaml`. As this action is always scheduled, I think we can remove the`workflow_dispatch` and `env` sections entirely.

Reply:
I think we can keep workflow_dispatch for now since we need to manually trigger the job or else we have to trigger the release action for this action to be triggered automatically

Reply:
Without the `env.` section this workflow_dispatch won't work as we're no longer setting `env.replace-version`.

Reply:
will remove workflow_dispatch since for this release we decided to do manually.